app/api/__init__.py

```diff
@@ -129,11 +129,12 @@
 api.route(TrackRelationship, 'track_sessions', '/tracks/<int:id>/relationships/sessions')
 api.route(TrackRelationship, 'track_event', '/tracks/<int:id>/relationships/event')
 
-# call_for_papers
-api.route(CallForPaperList, 'call_for_paper_list', '/events/<int:event_id>/call-for-papers',
-          '/events/<identifier>/call-for-papers')
-api.route(CallForPaperDetail, 'call_for_paper_detail', '/call-for-papers/<int:id>')
-api.route(CallForPaperRelationship, 'call_for_paper_event', '/call-for-papers/<int:id>/relationships/event')
+# speakers_calls
+api.route(SpeakersCallList, 'speakers_calls_list', '/events/<int:event_id>/speakers-calls')
```

Comment:
For home page, the data is got from events API's cfs relation. We don't directly get a list of cfs